hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/health/HealthCheckScheduler.java

```diff
@@ -0,0 +1,40 @@
+package pl.allegro.tech.hermes.management.domain.health;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import pl.allegro.tech.hermes.management.domain.mode.ModeService;
+import pl.allegro.tech.hermes.management.infrastructure.zookeeper.ZookeeperClientManager;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class HealthCheckScheduler {
+
+    private final ZookeeperClientManager zookeeperClientManager;
+    private final String healthCheckPath;
+    private final HealthCheckTask healthCheckTask;
+    private final Long period;
+    private final ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
```

Comment:
Please change default thread name to something more meaningful in this context. Example: https://github.com/allegro/hermes/blob/master/hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/message/undelivered/UndeliveredMessageLogPersister.java#L25

Reply:
Done.